buildme

```diff
@@ -1,53 +1,30 @@
 #!/bin/bash
-BUILDTYPE=Release
-ARCH=$(uname -m)
-ARM64=OFF
-CMAKE_TOOLCHAIN_FILE=../makefiles/cmake/toolchains/arm-linux-gnueabihf.cmake
 
-if [ "$1" = "--debug" ]; then
-	BUILDTYPE=Debug
-	shift
-fi
+CMAKE_BUILD_TYPE=Release
+CMAKE_TOOLCHAIN_FILE=/home/andreas/Code/Android/SDK/ndk/21.3.6528147/build/cmake/android.toolchain.cmake
```

Comment:
@AndistormEm This shouldn't be hardcoded right ?

Reply:
I adopted it from our old build script, I don't think everyone has the ndk at the same place, and I don't know anything about an entry in the environment path.